### DISCUSSION TITLE
Multilingual website

### DISCUSSION THREAD

**Discussion by mzilhao** [quarto-dev/quarto-cli#2876] [Q&A]
What's the best way to create a website that allows for display in different languages? Something with a language-switcher, for instance, similar to what hugo allows: https://gohugo.io/content-management/multilingual/ ?

**Comment by mcanouil** [comment]
Hi, I suggest you read and follow #275

**Reply by andresrcs** [reply]
The workaround is too technical and involved and since quarto websites are supposed to provide the same functionality offered by `blogdown` I think a more straightforward solution would be a reasonable request.

**Reply by cderv** [reply]
The idea is to subscribe to #275 which is still opened because something that needs to be done. 

Currently , workaround is not straitghfoward I agree.

**Reply by mcanouil** [reply]
A "workaround" can't really be "straightforward" by definition 😂

@cderv Maybe lock this discussion with a comment pointing to the issue? 🤔
